client/asset/btc/rpcclient.go

```diff
@@ -495,10 +495,31 @@ func (wc *rpcClient) listUnspent() ([]*ListUnspentResult, error) {
 	return unspents, wc.call(methodListUnspent, anylist{uint8(0)}, &unspents)
 }
 
+// lockAllUnspent unlocks all locked outputs for spending again. If unlock is
+// true btc will return 'true' if successful. If unlock is false btc will also
+// return 'true' but do nothing so a warning is issued below.
+func (wc *rpcClient) lockAllUnspent(unlock bool) error {
+	if !unlock {
+		wc.log.Warn("attempting to lock all unspent outputs 'lockunspent false'")
+	}
+	var rpcReturn bool
+	err := wc.call(methodLockUnspent, anylist{unlock}, &rpcReturn)
+	if err != nil {
+		return fmt.Errorf("'lockunspent %v': %v: ", unlock, err)
+	}
+	if !rpcReturn {
+		return fmt.Errorf("'lockunspent %v' unsuccessful: %v: RPC returns %v", unlock, err, rpcReturn)
```

Comment:
`err` is always `nil` here so I don't see a reason to include it's value in the message